users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -122,19 +128,28 @@ def run_nn_args(nn_args, report_args_collection, dev_corpora, report_name="", re
             "phon_history_length": 0,
             "phon_future_length": 0,
         },
-        scorer_info=score_info,
+        scorer_info=None,
         report=Report(
             columns_start=["train_name"],
             columns_end=["lm_scale", "prior_scale", "sub", "del", "ins", "wer"],
         ),
     )
+    ctc_nn_system._set_scorer("hub5e00", score_info_hub5e00)
+    ctc_nn_system._set_scorer("hub5e01", score_info_hub5e01)
     ctc_nn_system.crp["hub5e00"].acoustic_model_config.allophones.add_from_lexicon = False
     ctc_nn_system.crp["hub5e00"].acoustic_model_config.allophones.add_all = True
     ctc_nn_system.crp["hub5e00"].acoustic_model_config.allophones.add_from_file = tk.Path(
         "/u/vieting/setups/swb/20230406_feat/dependencies/allophones_blank",
         hash_overwrite="SWB_ALLOPHONE_FILE_WEI_BLANK",
         cached=True,
     )
+    ctc_nn_system.crp["hub5e01"].acoustic_model_config.allophones.add_from_lexicon = False
```

Comment:
I'd rather copy the allophone config from hub5e00 instead of redefining it here including the path.